fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/AvatarPie.kt

```diff
@@ -0,0 +1,186 @@
+        }
+    }
+}
+
+@Composable
+private fun RenderTwoSlices(
+    avatarSize: Dp, slicedAvatarDimen: Dp, group: Group, size: AvatarSize
+) {
+    Row(
+        modifier = Modifier
+            .requiredSize(avatarSize)
+            .clip(CircleShape)
+    ) {
+        SlicedAvatar(
+            group.members[0],
+            slicedAvatarSize = getSlicedAvatarSize(avatarSize),
+            width = slicedAvatarDimen,
+            modifier = Modifier
+                .height(avatarSize)
+                .width(slicedAvatarDimen),
+            size = size
+        )
+        AddVerticalSpacer()
+        SlicedAvatar(
+            group.members[1],
+            slicedAvatarSize = getSlicedAvatarSize(avatarSize),
+            width = slicedAvatarDimen,
+            modifier = Modifier
+                .height(avatarSize)
+                .width(slicedAvatarDimen),
+            size = size
+        )
+    }
+}
+
+@Composable
+private fun RenderThreeSlices(
+    avatarSize: Dp, slicedAvatarDimen: Dp, group: Group, size: AvatarSize
+) {
+    Row(
+        modifier = Modifier
+            .requiredSize(avatarSize)
+            .clip(CircleShape)
+    ) {
+        SlicedAvatar(
+            group.members[0],
+            slicedAvatarSize = getSlicedAvatarSize(avatarSize),
+            width = slicedAvatarDimen,
+            modifier = Modifier
+                .height(avatarSize)
+                .width(slicedAvatarDimen)
+                .align(Alignment.CenterVertically),
+            size = size
+        )
+        AddVerticalSpacer()
+        Column(
+            modifier = Modifier
+                .height(avatarSize)
+                .width(slicedAvatarDimen),
+        ) {
+            SlicedAvatar(
+                group.members[1],
+                slicedAvatarSize = getSlicedAvatarSize(slicedAvatarDimen),
+                width = slicedAvatarDimen,
+                modifier = Modifier
+                    .height(slicedAvatarDimen)
+                    .width(slicedAvatarDimen),
+                size = size
+            )
+            AddHorizontalSpacer()
+            SlicedAvatar(
+                group.members[2],
+                slicedAvatarSize = getSlicedAvatarSize(slicedAvatarDimen),
+                width = slicedAvatarDimen,
+                modifier = Modifier
+                    .height(slicedAvatarDimen)
+                    .width(slicedAvatarDimen),
+                size = size
+            )
+        }
+
+    }
+}
+
+@Composable
+private fun AddVerticalSpacer() {
+    Spacer(
+        modifier = Modifier
+            .background(color = Color.White)
+            .fillMaxHeight()
+            .width(SPACER_SIZE)
+    )
+}
+
+@Composable
+private fun AddHorizontalSpacer() {
+    Spacer(
+        modifier = Modifier
+            .background(color = Color.White)
+            .fillMaxWidth()
+            .height(SPACER_SIZE)
+    )
+}
+
+
+private fun getSlicedAvatarSize(size: Dp): SlicedAvatarSize {
```

Comment:
why is this required. Are these values fluent?

Reply:
removed this